Add details about FormData with a simple object (#1019)
grafana/k6-docs: src/data/markdown/docs/05 Examples/01 Examples/09 data-uploads.md

  will appear last in the request under the `text` form field.
- It's required to specify the multipart boundary in the `Content-Type` header,
  so you must assemble the header manually as shown.

> - Blob is not supported or implemented. The same functionality can be achieved using

Reviewer: ```suggestion
- Blob is not supported or implemented. For the same functionality, use
```
